internal/app/handlers.go

```diff
@@ -51,10 +57,13 @@ func (app *Application) getCodeSystemByID(w http.ResponseWriter, r *http.Request
 	}
 
 	var codeSystem *xo.CodeSystem
-	if id_type == "oid" {
+	switch id_type {
+	case Oid:
 		codeSystem, err = rp.GetCodeSystemByOID(r.Context(), id)
-	} else {
+	case Id:
 		codeSystem, err = rp.GetCodeSystemByID(r.Context(), id)
+	case Unknown:
+		customErrors.ServerError(w, r, customErrors.ErrInvalidId, app.logger)
```

Comment:
I made this a server error since the only usage of unknown should be along side an error that's. handled about 10 lines up. I'm not sure if this is the best pattern though

Reply:
@katyasoup could you take a look at this when you're back? See if we should revisit this pattern and what is the right thing to do?

@mcmcgrath13 for now, make this panic instead